Source/WebCore/html/canvas/OffscreenCanvasRenderingContext2D.cpp

```diff
@@ -145,16 +145,16 @@ RefPtr<ImageBuffer> OffscreenCanvasRenderingContext2D::transferToImageBuffer()
 
 CanvasDirection OffscreenCanvasRenderingContext2D::direction() const
 {
-    // FIXME: What should we do about inherit here?
     switch (state().direction) {
     case Direction::Inherit:
+        return Direction::Inherit;
     case Direction::Ltr:
         return Direction::Ltr;
     case Direction::Rtl:
         return Direction::Rtl;
     }
     ASSERT_NOT_REACHED();
-    return Direction::Ltr;
+    return Direction::Inherit;
```

Comment:
What is the value in the specification when not defined?

Reply:
But here it says in step 5:
https://html.spec.whatwg.org/multipage/canvas.html#dom-offscreencanvas

> Set [this](https://webidl.spec.whatwg.org/#this)'s [inherited direction](https://html.spec.whatwg.org/multipage/canvas.html#offscreencanvas-inherited-direction) to "ltr".

so I have doubts now.